source/_templates/installations/basic/elastic/common/configure_kibana.rst

```diff
@@ -15,7 +15,7 @@ Edit the ``/etc/kibana/kibana.yml`` file:
 
 Values to be replaced:
 
-- ``<kibana_ip>``: by default, Kibana only listens on the loopback interface (localhost), which means that it can be only accessed from the same machine. To access Kibana from the outside it may be configured to listen on its network IP address by replacing ``kibana_ip`` with Kibana's host IP address.
+- ``<kibana_ip>``: by default, Kibana only listens on the loopback interface (localhost), which means that it can be only accessed from the same machine. To access Kibana from the outside, it may be configured to listen on its network IP address by replacing ``kibana_ip`` with Kibana's host IP address.
```

Comment:
```suggestion
- ``<kibana_ip>``: by default, Kibana only listens on the loopback interface (localhost), which means that it can be only accessed from the same machine. To access Kibana from the outside, it may be configured to listen on its network IP address by replacing ``kibana_ip`` with Kibana host IP address.
```